app/models/deferred/visitor.rb

```diff
@@ -0,0 +1,4 @@
+module Deferred
+  class Visitor < ::Visitor
+  end
```

Comment:
I'm tempted to write something like the [10,000 year nuclear waste warning](http://www.wipp.energy.gov/picsprog/articles/wipp%20exhibit%20message%20to%2012,000%20a_d.htm):

> This place is not a place of honor.
> No highly esteemed deed is commemorated here.
> Nothing valued is here.
> This place is a message and part of a system of messages.
> Pay attention to it!
> Sending this message was important to us.
> We considered ourselves to be a powerful culture.